src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+@import "../base/base";
+@import "../abstracts/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  width: calc(#{$cell-dimensions + $gap + $padding} * 7);
+
+  &__day {
+    background-color: $cell-color;
+    font-size: $font;
+    width: $cell-dimensions;
+    height: $cell-dimensions;
+    box-sizing: border-box;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    transition: background-color 0.5s;
```

Comment:
please make sure to animate transform, because right now only bg color is animated